django_nameko/rpc.py

```diff
@@ -93,8 +97,9 @@ def __exit__(self, exc_type, exc_value, traceback, **kwargs):
                             for key in self._rpc._worker_ctx.data.keys():
```

Comment:
it's not clear what this code do, and how to cover it with unittests.

Reply:
this part of the code is for checking context data of the worker after RPC is complete and make sure it is rever back to original pool context data

Reply:
can you give me a snipet of code where the worker context is altered ? I did not succeed in doing it in the tests.